Playwright.Dockerfile

```
@@ -1,13 +1,14 @@
-FROM mcr.microsoft.com/playwright:v1.49.1-jammy
+FROM mcr.microsoft.com/playwright:v1.51.0-jammy
 
-RUN apt-get update && apt-get install -y python3 make g++ \
-    && npm install -g pnpm \
-    && apt-get clean
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)